src/app/ues/[code]/comments/[commentId]/page.tsx

```diff
@@ -0,0 +1,123 @@
+  const api = useAPI();
+  if (!comment || !ue || !user) {
+    return;
+  }
+  return (
+    <div className={styles.page}>
+      <h1>
+        {comment.isAnonymous
+          ? t('ues:detailed.comments.resume.anonymous', {
+              ue: ue.code,
+              semester: comment.semester.code,
+              date: comment.createdAt.toLocaleDateString(),
+            })
+          : t('ues:detailed.comments.resume', {
+              authorFirstName: comment.author.firstName,
+              authorLastName: comment.author.lastName,
+              ue: ue.code,
+              semester: comment.semester.code,
+              date: comment.createdAt.toLocaleDateString(),
+            })}
+      </h1>
+      {comment.updatedAt && (
+        <p className={styles.updateDate}>
+          {t('ues:detailed.comments.updatedAt', { date: comment.updatedAt.toLocaleDateString() })}
+        </p>
+      )}
+      <p className={styles.body}>{comment.body}</p>
+      <div className={styles.comments}>
+        {comment.answers.map((answer, i) => (
+          <div key={answer.id} className={styles.comment}>
+            <p className={styles.author}>
+              {answer.author
+                ? `${answer.author.firstName} ${answer.author.lastName}`
+                : t('ues:detailed.comments.author.deleted')}
+            </p>
+            <p className={styles.date}>
+              {t('ues:detailed.comments.writtenDate', { date: answer.createdAt.toLocaleDateString() })}
+            </p>
+            <EditableText
+              className={styles.body}
+              text={answer.body}
+              EditingFooter={CommentEditorFooter(
+                answer.body,
+                async (body) => {
+                  const newAnswer = await editCommentReply(api, answer.id, body).toPromise();
+                  if (!newAnswer) return false;
+                  setComment({
+                    ...comment,
+                    answers: [...comment.answers.slice(0, i), newAnswer, ...comment.answers.slice(i + 1)],
+                  });
+                  return true;
+                },
+                t,
+              )}
+              enabled={answer.author.id === user.id}
+            />
+          </div>
+        ))}
+      </div>
+      <h2 className={styles.answerTitle}>{t('ues:detailed.comments.answers.answerTitle')}</h2>
+      <TextArea
+        className={styles.input}
+        onChange={setAnswer}
+        value={answer}
+        placeholder={t('ues:detailed.comments.answers.answerEntry')}
+      />
+      <div className={styles.buttonWrapper}>
+        <Button
+          className={styles.button}
+          onClick={() =>
+            sendCommentReply(api, params.commentId, answer)
+              .toPromise()
+              .then((answer) => {
+                if (!answer) return;
+                setAnswer('');
+                setComment({ ...comment, answers: [...(comment?.answers ?? []), answer] });
+              })
+          }>
```

Comment:
Ptet on peut utiliser un async ?
```suggestion
          onClick={async () =>
            const answer = await sendCommentReply(api, params.commentId, answer)
              .toPromise();
            if (!answer) return;
            setAnswer('');
            setComment({ ...comment, answers: [...(comment?.answers ?? []), answer] });
          }>
```